### ISSUE TITLE
How the #&@%# did I break this?

### ISSUE THREAD

**Issue by ericsink** [ericsink/SQLitePCL.raw#436]
Yeah it seems the issue is in `SQLitePCLRaw.lib.e_sqlite3`.

For version 2.0.4 it has the required folders:
![image](https://user-images.githubusercontent.com/249088/129785598-a9ee1a92-2af0-4855-9f3f-c035967faf9b.png)

But for the latest 2.0.5 preview it doesn't:
![image](https://user-images.githubusercontent.com/249088/129785648-5eb6fb5b-a953-4de6-a5af-9143a71963b6.png)

_Originally posted by @DamianEdwards in https://github.com/dotnet/efcore/issues/24198#issuecomment-900558428_

**Comment by stefc** [comment]
Did you push a new package on NuGet for this fix ? Still not seeing netstandard 2.0 

![image](https://user-images.githubusercontent.com/464696/129931838-180e6292-40a6-4a7d-8d96-d9d26264ad57.png)

**Comment by ericsink** [comment]
No, this doesn't appear yet in any package on nuget.  I just made the fix in the code.  The next -pre will have the fix.
